seed/static/seed/js/controllers/inventory_list_beta_controller.js

```diff
@@ -1108,33 +1064,57 @@ angular.module('BE.seed.controller.inventory_list_beta', [])
 
       $scope.model_actions = 'none';
       elSelectActions = document.getElementById('select-actions');
-      $scope.run_action = function () {
+      $scope.run_action = function (viewIds = []) {
+        let selectedViewIds = [];
+
+        // was the function called with a list of ids?
+        if (viewIds.length > 0) {
+          selectedViewIds = viewIds;
+
+        // if it appears everything selected, only get the full set of ids...
+        } else if ($scope.selectedCount == $scope.inventory_pagination.total) {
+          selectedViewIds = [];
+
+          $http.post('/api/v3/properties/filter/', {}, {
+            params: {
+              organization_id: $scope.organization.org_id,
+              ids_only: true
+            }
```

Comment:
As written, it looks like this will not appliy the user's filters and sorting, so I think we should add that here.
It might be better to use the inventory_service for the call as well instead of using $http